sdk/iOS/src/MSTableOperation.h

```diff
@@ -28,13 +28,13 @@ typedef NS_OPTIONS(NSUInteger, MSTableOperationTypes) {
 @property (nonatomic, readonly) MSTableOperationTypes type;
 
 /// The name of the table associated with the item
-@property (nonatomic, copy, readonly) NSString *tableName;
+@property (nonatomic, copy, readonly, nonnull) NSString *tableName;
 
 /// The Id of the item the operation should run on.
-@property (nonatomic, copy, readonly) NSString *itemId;
+@property (nonatomic, copy, readonly, nullable) NSString *itemId;
```

Comment:
why is this one nullable, I'd have assumed all ops have itemIds and its item that is the nullable one.